src/EthereumHelpers.js

```diff
@@ -129,35 +158,21 @@ async function sendSafely(boundContractMethod, sendParams, forceSend) {
     const gasEstimate = await boundContractMethod.estimateGas({ ...sendParams })
 
     return boundContractMethod.send({
+      from: "", // FIXME Need systemic handling of default from address.
       ...sendParams,
       gas: gasEstimate
     })
   } catch (exception) {
-    // If we're not forcibly sending, try to resolve the true error by using
-    // `call`.
+    // For an always failing transaction, if forceSend is set, send it anyway.
     if (
       exception.message &&
-      exception.message.match(/always failing transaction/)
+      exception.message.match(/always failing transaction/) &&
+      forceSend
     ) {
-      let callResult
```

Comment:
Just curious about why we're dropping this logic? Is it because it's not actually doing anything (i.e. "FIXME Something more is needed here to properly resolve this error...)?

Reply:
Ah yes, I forgot to flag this at the PR level! Short version here is---it wasn't doing anything. `handleRevert` bubbles the error up without this extra dance, and before `handleRevert` was set, this didn't actually do what it was supposed to---though I'm not sure why :/

Reply:
Aha. Thanks for clarifying!